packages/mobile/src/components/suggested-tracks/SuggestedCollectionTracks.tsx

```diff
@@ -0,0 +1,62 @@
+import type { ID } from '@audius/common'
+import {
+  useGetCurrentUserId,
+  useGetPlaylistById,
+  useGetSuggestedAlbumTracks,
+  useGetSuggestedPlaylistTracks
+} from '@audius/common'
+
+import { SuggestedTracks } from './components/SuggestedTracks'
+
+type SuggestedCollectionTracksProps = {
+  collectionId: ID
+}
+
+const SuggestedAlbumTracks = (props: SuggestedCollectionTracksProps) => {
+  const { collectionId } = props
+  const { suggestedTracks, onRefresh, onAddTrack, isRefreshing } =
+    useGetSuggestedAlbumTracks(collectionId)
+
+  return (
+    <SuggestedTracks
+      collectionId={collectionId}
+      suggestedTracks={suggestedTracks}
+      onRefresh={onRefresh}
+      onAddTrack={onAddTrack}
+      isRefreshing={isRefreshing}
+    />
+  )
+}
+
+const SuggestedPlaylistTracks = (props: SuggestedCollectionTracksProps) => {
+  const { collectionId } = props
+  const { suggestedTracks, onRefresh, onAddTrack, isRefreshing } =
+    useGetSuggestedPlaylistTracks(collectionId)
+
+  return (
+    <SuggestedTracks
+      collectionId={collectionId}
+      suggestedTracks={suggestedTracks}
+      onRefresh={onRefresh}
+      onAddTrack={onAddTrack}
+      isRefreshing={isRefreshing}
+    />
+  )
+}
+
+export const SuggestedCollectionTracks = (
+  props: SuggestedCollectionTracksProps
```

Comment:
I see so just easier to use this one component in both places and pass a prop in?

Reply:
You don't even need a prop. `collectionId` is sufficient to differentiate between collectionTypes since we're selecting the collection anyway.